module-cheatsheets/intro-to-R-tidyverse-cheatsheet.md

```diff
@@ -15,23 +15,30 @@ Read the Base `R` documentation [**here**](https://www.rdocumentation.org/packag
-| Base `R`          | [`as.factor(x)`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/factor)       | As Factor             | Coerces object x into a factor (which is used to represent categorical data). This function can be used to coerce object `x` into other data types, i.e., `as.character`, `as.data.frame`, `as.matrix`, etc. |
-| Base `R`          | [`dim(x)`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/dim)                | Dimensions summary    | Returns the dimensions of object `x`.                                                                                                                                                               |
+| Base `R`          | [`factor(x)` or `as.factor(x)`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/factor)       | Factor             | Coerces object x into a factor (which is used to represent categorical data). This function can be used to coerce object `x` into other data types, i.e., `as.character`, `as.numeric`, `as.data.frame`, `as.matrix`, etc. |
+| Base `R`          | [`summary(x)`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/summary)                | Object summary    | Returns a summary of the values in object `x`.                                                                                                                                                               |
 | Base `R`          | [`data.frame()`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/data.frame)   | Data Frame            | Creates a data.frame where the named arguments will be the same length                                                                                             |
-| Base `R`          | <a href = "https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/Extract">`[[x]]`</a>   |  Double bracket       | Extracts `x` from a nested list                                |
 | Base `R`          | [`sessionInfo()`](https://www.rdocumentation.org/packages/utils/versions/3.5.1/topics/sessionInfo) | Session Information   | Returns the R version information, the OS, and the attacked packages in the current R session                                                                                                 |
-| Base `R`          | [`getwd()`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/getwd)             | Get working directory | Finds the current working directory.                                                                                                                                                              |
-| Base `R`          | [`setwd()`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/getwd)             | Set working directory | Changes the current working directory.                                                                                                                                                            |
```

Comment:
We may not be using this explictly in the intro, but I think it should stay in the docs.